src/Command/DoctrineDecryptDatabaseCommand.php

```diff
@@ -94,65 +84,78 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
 
         // Start decrypting database
+        $_ENV['DOCTRINE_SKIP_ENCRYPT'] = true;
```

Comment:
I would not use environment variables for this, for multiple reasons.

One being that a user could modify how the code of the `Encrypt`-service works by setting environment-variables, another that is a variant of global variables, which are kind of a bad practice in my opinion.

I would simply add boolean properties to your `Encrypt`-service with public setters which you call from here.